### DISCUSSION TITLE
cant run on win11

### DISCUSSION THREAD

**Discussion by kk5568** [t1m0thyj/WinDynamicDesktop#369] [General]
it crash when I select the theme

**Comment by t1m0thyj** [comment]
Other users are able to run the app fine on Win11. Could you please provide more details, like a screenshot of the error, or log files (WinDynamicDesktop.exe.log and debug.log)?

**Reply by kk5568** [reply]
https://user-images.githubusercontent.com/74945328/135082994-bb5d7447-8116-4223-90f7-c4b561a6e38b.mp4
Does this work?

**Reply by t1m0thyj** [reply]
I don't see the app crashing in that video, just that it doesn't show the Select Theme dialog as expected.

I assume that this issue persists after restarting your machine? If the app is still crashing, please check if there is a log file where WDD is installed and attach it here (see here for [instructions](https://github.com/t1m0thyj/WinDynamicDesktop/wiki/Troubleshooting#finding-where-app-is-installed)).
